VMBackup/main/common.py

```diff
@@ -42,7 +42,6 @@ class CommonVariables:
     statusBlobUploadError = 'statusBlobUploadError'
     TempStatusFileName = 'tempStatusFile.status'
     onlyLocalFilesystems = 'onlyLocalFilesystems'
-
```

Comment:
There seems to be no change in the file. I think this file changes can be reverted.